packages/commons-test/src/setupTestContainersVitestGlobal.ts

```diff
@@ -106,10 +122,23 @@ export function setupTestContainersVitestGlobal() {
       provide("fileManagerConfig", fileManagerConfig.data);
     }
 
+    if (emailManagerConfig.success) {
+      startedMailpitContainer = await mailpitContainer().start();
+      emailManagerConfig.data.emailManagerPort =
+        startedMailpitContainer.getMappedPort(TEST_MAILPIT_SMTP_PORT);
+      emailManagerConfig.data.emailManagerHTTPPort =
+        startedMailpitContainer.getMappedPort(TEST_MAILPIT_HTTP_PORT);
+      emailManagerConfig.data.emailManagerHost =
+        startedMailpitContainer.getHost();
+      emailManagerConfig.data.emailManagerSender = TEST_EMAIL_MANAGER_SENDER;
```

Comment:
Setting `emailManagerSender` is probably not needed, it's unrelated to the test container, the tests should be able to read it from the `.env` file so if you remove this everything should keep working anyways